docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
 
   # postgres
   postgres:
-    image: "postgres:9.6.1"
+    image: "mdillon/postgis:9.6"
```

Comment:
I guess somebody else can easily create a PR for the feature with specific extension and put another docker image with that extension, but without gis let's say. So better solution would be to have a base postgres image with all modifications applied

Reply:
Agreed. `mdillon/postgis` is `postgres` + PostGIS. Is this worth punting on until someone else needs to add another extension? (We can add a note/apology here ;-)

Reply:
haha I yeah... I hope it won't be me 😆